Templates/AbstractAjaxTemplate/Elements/JqueryButtonTrait.php

```diff
@@ -14,7 +14,7 @@ protected function buildJsInputRefresh(Button $widget, $input_element)
     {
         $js = ($widget->getRefreshInput() && $input_element->buildJsRefresh() ? $input_element->buildJsRefresh(true) . ";" : "");
         if ($link = $widget->getRefreshWidgetLink()) {
-            if ($link->getPageId() == $widget->getPageId() && $linked_element = $this->getTemplate()->getElement($link->getWidget())) {
+            if ($link->getPageAlias() == $widget->getPageAlias() && $linked_element = $this->getTemplate()->getElement($link->getWidget())) {
```

Comment:
Diese vergleiche der ID sind mit dem Replacement-Mechanismus nicht mehr sicher. Ersetze sie bitte durch $link->getPage()->is($widget->getPage()).